### PR TITLE
add headers and intro text to form templates

### PR DESCRIPTION
**Pull request by carodew**
I also deleted a duplicate SSN form.

### CODE REVIEW

**Comment by carodew** [comment]
I need to migrate some content for some of the form templates in today but I don't want to create a new branch to avoid merge conflicts. Could someone review this and merge or at least hold off and I can add the additional content in here? TY! @msecret @maya @vgvg

File: _components/form-templates.md [hunk 1]
```diff
@@ -175,61 +178,12 @@ lead: Patterns for some of the most commonly used forms on government websites.
   </div>
 </div>
 
-<div class="preview">
```
Comment (maya) [inline _components/form-templates.md]: @carodew for some reason I'm not seeing a duplicate SSN form on staging. I'm seeing just one.

File: _components/form-templates.md [hunk 2]
```diff
@@ -352,8 +307,8 @@ lead: Patterns for some of the most commonly used forms on government websites.
     </ul>
   </div>
 </div>
-
-<div class="preview" id="ssn-form">
+<h2 class="usa-heading" id="ssn-form">Social security number form</h2>
```
Comment (maya) [inline _components/form-templates.md]: Is it OK we're using sentence case for the heading but capitalizing the legend?

File: _components/form-templates.md (hunk 1 above)
Comment (carodew) [inline _components/form-templates.md]: It's not in the side nav, but it is between the address form and the login form.

Comment (maya) [inline _components/form-templates.md]: Ok I see it now :+1:

File: _components/form-templates.md (hunk 2 above)
Comment (carodew) [inline _components/form-templates.md]: Well, generally we should be using sentence case as per the 18F content style guide (https://pages.18f.gov/content-guide/capitalization/) but your question made me stop to think about SSN being a proper noun. So far the writing lab seems to think Social Security Number should be capitalized.

Comment (carodew) [inline _components/form-templates.md]: ok, fixed in e0d2a6c

**Comment by maya** [comment]
lgtm

**Pull request merged**